ERCS/erc-798.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: EIP-798
+title: ERC-798, EIP-721 Guarantee Extension
```

Comment:
```suggestion
title: ERC-721 Guarantee Extension
```
Don't include ERC number in title

Reply:
Thanks a lot,it has been modified as suggested.